token.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+const uuid = require('node-uuid');
+const log = require('./log.js');
+
+let insertToken;
+let consumeToken;
+let deleteOldTokens;
+let db;
```

Comment:
Due to these top level variables this module now relies on the first script to use `token.js` to call `setDb` first. I think it's better to instead export a constructor function that takes a db object and returns the rest of the interface. This makes sure the db object & queries are always sets, and additionally makes the module reusable (and thus testable).